crates/bevy_sprite/src/mesh2d/mesh.rs

```diff
@@ -39,7 +39,7 @@ use crate::Material2dBindGroupId;
 ///
 /// It wraps a [`Handle<Mesh>`] to differentiate from the 3d pipelines which use the handles directly as components
 #[derive(Default, Clone, Component, Debug, Reflect, PartialEq, Eq)]
-#[reflect(Component)]
+#[reflect(Default, Component)]
 pub struct Mesh2dHandle(pub Handle<Mesh>);
```

Comment:
Just checking, but is this type registered in one of these render plugins? If not, could we register it in one?

Reply:
`SpritePlugin` registers it.